packages/core/interfaces/src/compilerSupport/aquaTypeDefinitions.ts

```diff
@@ -25,6 +25,11 @@ export type SimpleTypes =
 
```

Comment:
Should this file be inside `aqua-api`?

Reply:
Yes, it should. Need to discuss with compiler team

Reply:
If you need to move stuff around - there is no problem. Just edit aqua-api and put what you need in there. I did a wrapper around the interface that scala-js produces - this can be improved, extended, preferably by us who know typescript, instead of guys why write the compiler